modules/configuration/src/test/java/org/apache/ignite/internal/configuration/notifications/ConfigurationListenerTest.java

```diff
@@ -915,4 +957,17 @@ void testGetConfigFromNotificationEventOnUpdate() throws Exception {
         
         configuration.elements().get(key).str().update(newVal).get(1, SECONDS);
     }
+
+    @Test
+    void listenerGetsNotifiedOnCommonPropertyChangeInPolymorphicBaseConfigUnderNoNamedConfig() throws Exception {
```

Comment:
The name is very complicated

Reply:
Well, the logic the test is verifying cannot be put in a couple of words (there are several conditions that comprise the context of the test). A test 'method' is a method only technically; in reality, it's a 'test case', and it seems good to capture 'what is being tested here' in its name.

If this does not convince you, how would you suggest to name it?

Reply:
The name should be simpler, and if an explanation is needed, then it should be added to the description of the method.

Reply:
This is not actually a name because nobody uses test methods for invocations. This is actually a description of a test. When you read a test report, you only see the 'method name' and not javadoc, so it seems more useful to put test case description into a name and not into a javadoc that you can only see in the code.

Reply:
Shortened it to `polymorphicParentFieldChangeNotificationHappens`. Would it be ok or should we shorten it further?